docs/index.asciidoc

```diff
@@ -59,4 +61,34 @@ actual encoding of the text and Logstash will convert it for you.
 
 For nxlog users, you may to set this to "CP1252".
 
-
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: does not use ECS-compatible field names
```

Comment:
Is it expected that, at this stage, the description of `ecs_compatibility` is the same in all plugins that support ECS? I noticed some differences from, for example, https://github.com/logstash-plugins/logstash-input-tcp/pull/165/files#diff-cae5619b3d18ec99c5ccd0a9f6de0c6d3f53343c64692444551a7d29da6863e7R158-R167

Reply:
yeah, isn't really the same in all plugins, usually includes smt slightly plugin specific see e.g. [CEF codec](https://github.com/logstash-plugins/logstash-codec-cef/blob/master/docs/index.asciidoc#ecs_compatibility)
this one has been :scissors: from the [JSON filter](https://github.com/logstash-plugins/logstash-filter-json/blob/master/docs/index.asciidoc#ecs_compatibility)

Reply:
Right but it seems that, for example, both v1 and v8 should be documented as valid settings as we're establishing the alias at the top of the plugin file.